lib/public/components/Log/logText.js

```diff
@@ -19,14 +19,17 @@ import { markdownDisplay } from '../common/markdown/markdown.js';
  *
  * @param {Log} log the log to display
  * @param {Boolean} isCollapsed an indicator as to whether the log is collapsed
+ * @param {Object} opts other opts
+ * @param {Partial<MarkdownBoxSize>} [opts.boxSize] markdown size definition
  * @return {Component} the log's text display
  */
-export const logText = (log, isCollapsed) => h(
+export const logText = (log, isCollapsed, opts = {}) => h(
     `div#log-id-${log.id}`,
     isCollapsed
         ? h(`.text-ellipsis.w-100#log-text-${log.id}`, log.text)
         : markdownDisplay(log.text, {
             classes: 'w-100',
             id: `log-text-${log.id}`,
-        }),
+            ...opts,
```

Comment:
if the only option you expect is `boxSize` you should not put this here?

Reply:
deleted